Sitemap/ChainProvider.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+use Symfony\Cmf\Bundle\SeoBundle\Model\UrlInformation;
+
+/**
+ * Accepts providers and merges the result of all providers into
+ * a combined list of UrlInformation
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@gmx.de>
+ */
+class ChainProvider implements UrlInformationProviderInterface
+{
+    /**
+     * @var array
+     */
+    private $sortedProviders = array();
+
+    /**
+     * {@inheritDoc}
+     */
+    public function addProvider(UrlInformationProviderInterface $provider, $priority = 0)
+    {
+        if (empty($this->sortedProviders[$priority])) {
+            $this->sortedProviders[$priority] = array();
+        }
+        $this->sortedProviders[$priority][] = $provider;
+    }
+
+    /**
+     * @return UrlInformationProviderInterface[]
+     */
+    private function getSortedProviders()
+    {
+        $sortedProviders = array();
+        ksort($this->sortedProviders);
+
+        foreach ($this->sortedProviders as $providers) {
+            $sortedProviders = array_merge($sortedProviders, $providers);
+        };
+        return$sortedProviders;
```

Comment:
there should be an empty line before return statements and there should be a space between the return statement and the expression.